cmd/zpool/zpool_main.c

```diff
@@ -4266,34 +4266,25 @@ get_columns(void)
 	return (columns);
 }
 
-int
-get_namewidth(zpool_handle_t *zhp, void *data)
+static int
+get_namewidth(zpool_handle_t *zhp, int *width, int flags, boolean_t verbose)
```

Comment:
1. What about having `get_namewidth()` return the width instead of having a separate `*width`?
2. Can you add a quick comment above the function deceleration:
`/* Return the length of the pool/vdev name column. */`

Reply:
Sure, after reworking things a little more I think that'll work.